book/content/continuous_integration/continuous_integration.md

```diff
@@ -2,10 +2,10 @@
 
 | Prerequisite | Importance | Notes |
 | -------------|------------|-------|
```

Comment:
This is so fab @ClareLiggins! I wonder if these should go in a different order though?

I'd guess command line, version control, environments then testing?

Reply:
@KirstieJane Thanks for the suggestions - I have made the changes to these now. Does the pull request need to be approved again to merge?